system/version.py

```diff
@@ -103,6 +108,9 @@ def tested_channel(self) -> bool:
   def release_channel(self) -> bool:
     return self.channel in RELEASE_BRANCHES
 
+  def canonical(self) -> str:
+    return f"{self.openpilot.version}-{self.openpilot.short_commit}-{self.openpilot.build_style}"
```

Comment:
I was thinking we use the full commit hash here

Reply:
`0.9.7-f24ddc5f507c2cc3f0176b4ec176b7ef953036f8-release.caidx`

done